subprojects/gradle/code-ownership/src/main/kotlin/com/avito/android/CodeOwnershipPlugin.kt

```diff
@@ -12,31 +13,33 @@ public class CodeOwnershipPlugin : Plugin<Project> {
     override fun apply(target: Project) {
         check(target.isRoot()) { "Code ownership plugin must be applied to the root project" }
 
-        val enabled = target.getBooleanProperty("avito.moduleOwnershipValidationEnabled", false)
+        val strictOwnership = target.getBooleanProperty("avito.ownership.strictOwnership", false)
 
         target.subprojects { subproject ->
```

Comment:
We have to avoid iterating through subprojects. 
It gives a performance penalty while configuring a project.

This is the client's responsibility to apply the plugin to all modules. It can use convention plugins.
Please, schedule a fix for this issue.

Reply:
MBS-12327